content/v2.0/views/action-integration.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Action integration
+order: 20
+---
+
+Hanami actions are designed to work seamlessly with Hanami views, with features like automatic view rendering and support for a context object that offers views access to the current request.
+
+### Automatic view rendering
+
+By convention, Hanami actions will automatically render a corresponding view, should that view exist.
+
+For example, this `Pages::Contact` action will render a `Pages::Contact` view:
+
+```ruby
+# app/actions/pages/contact.rb
+
+module Bookshelf
+  module Actions
+    module Pages
+      class Contact < Bookshelf::Action
+        def handle(*, response)
```

Comment:
:thought_balloon: Is it possible to automatically render your view but still provide custom arguments to your view? For example, something like this:

``` ruby
def handle(*, response) = response[contact: repository.find[:id]]
```

I couldn't make this work so wanted to ask. I'm guessing the answer is no since anything beyond default behavior needs to be specific?